creator-node/docker-compose/development.env

```diff
@@ -30,10 +30,12 @@ serviceCountry=
 # ipfsClusterPort=0
 
 # wallet information required on all environments
-# delegateOwnerWallet=
-# delegatePrivateKey=
+# delegateOwnerWallet=0x1eC723075E67a1a2B6969dC5CfF0C6793cb36D25
+# delegatePrivateKey=0xdb527e4d4a2412a443c17e1666764d3bba43e89e61129a35f9abc337ec170a5d
 spOwnerWalletIndex=1
 
+creatorNodeEndpoint=http://docker.for.mac.localhost:4000
```

Comment:
i'll go ahead and remove these endpoints too? @dmanjunath @SidSethi

Reply:
yes please! @vicky-g. reason for asking to remove this is because this property has to do with cnode self identity which is something which is a part of the auth work

Reply:
yep, all mentions of it can / should be removed